docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json

```diff
@@ -56,6 +56,14 @@
       "description": "The declaration of conditions used to dynamically enable features.",
       "required": [],
       "properties": {
+        "requirement_type": {
+          "$id": "#/properties/conditions/properties/requirement_type",
+          "type": "string",
+          "title": "Requirement Type",
+          "description": "Determines whether any or all registered client filters must be enabled for the feature to be considered enabled.",
+          "enum": [ "Any", "All" ],
```

Comment:
I see a few other small arrays, none inlined though.

Reply:
I realize that I inlined it in my suggestion, whoops. Would still recommend not to do so since it's not done elsewhere in the file.